### PR TITLE
Avoid using dev-deps in post-install script

### PR DESCRIPTION
**Pull request by AMoo-Miki**
Use native methods in post-install

Signed-off-by: Miki <miki@amazon.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: scripts/postinstall.js [hunk 1]
```diff
@@ -21,23 +21,26 @@ if (fs.existsSync(targetFilePath)) {
...
-  shell.cp('-fR', 'src/themes/oui-cascadia/overrides', 'src/themes/eui-amsterdam/overrides');
-  shell.cp('-fR', 'src/themes/oui-cascadia/oui_cascadia_colors_dark.scss', 'src/themes/eui-amsterdam/eui_amsterdam_colors_dark.scss');
-  shell.cp('-fR', 'src/themes/oui-cascadia/oui_cascadia_colors_light.scss', 'src/themes/eui-amsterdam/eui_amsterdam_colors_light.scss');
-  shell.cp('-fR', 'src/themes/oui-cascadia/oui_cascadia_globals.scss', 'src/themes/eui-amsterdam/eui_amsterdam_globals.scss');
+  // Clean up and recreate the folders
+  fs.rmSync('src/themes/eui', { recursive: true, force: true });
+  fs.rmSync('src/themes/eui-amsterdam', { recursive: true, force: true });
+
+  const copyDirectory = (src, dest) => {
+    fs.mkdirSync(dest, { recursive: true });
```
Comment (ashwin-pc) [inline scripts/postinstall.js:32]: Why sync?

File: scripts/postinstall.js [hunk 2]
```diff
@@ -21,23 +21,26 @@ if (fs.existsSync(targetFilePath)) {
...
+
+    for (const entry of fs.readdirSync(src, { withFileTypes: true })) {
+      const srcItem = path.join(src, entry.name);
+      const destItem = path.join(dest, entry.name);
+
+      if (entry.isDirectory()) copyDirectory(srcItem, destItem);
+      else fs.copyFileSync(srcItem, destItem);
```
Comment (ashwin-pc) [inline scripts/postinstall.js:39]: Again, why sync?

**Review by ashwin-pc — COMMENTED** [review]
Nit comments but looks good otherwise

**Review by ashwin-pc — APPROVED** [review]

File: scripts/postinstall.js (hunk 2 above)
Comment (AMoo-Miki) [reply scripts/postinstall.js:39]: No good reason. Just because existing code was sync.
BTW, these are just temporary while we alias.

**Review by bandinib-amzn — APPROVED** [review]

**Review by ashwin-pc — APPROVED** [review]

**Pull request merged**